openedx_tagging/core/tagging/rest_api/v1/serializers.py

```diff
@@ -61,22 +63,60 @@ class ObjectTagListQueryParamsSerializer(serializers.Serializer):  # pylint: dis
     )
 
 
-class ObjectTagSerializer(serializers.ModelSerializer):
+class ObjectTagMinimalSerializer(serializers.ModelSerializer):
     """
-    Serializer for the ObjectTag model.
+    Minimal serializer for the ObjectTag model.
     """
 
     class Meta:
         model = ObjectTag
-        fields = [
+        fields = ["value", "lineage"]
+
+    lineage = serializers.ListField(child=serializers.CharField(), source="get_lineage")
+
+
+class ObjectTagSerializer(ObjectTagMinimalSerializer):
+    """
+    Serializer for the ObjectTag model.
+    """
+    class Meta:
```

Comment:
Need to provide Meta.model, otherwise visiting http://127.0.0.1:8000/tagging/rest_api/v1/object_tags/abc/ throws an exception `Class ObjectTagSerializer missing "Meta.model" attribute`

```suggestion
    class Meta:
        model = ObjectTag
```

Reply:
FYI even with the `Meta.model` set, I still get an error in the DRF UI.
But I'm unable to reproduce this issue in tests, and so maybe we don't care?

It happens regardless of whether there are any tags applied to the requesting object.

```
AttributeError at /tagging/rest_api/v1/object_tags/abc/

Got AttributeError when attempting to get a value for field `lineage` on serializer `ObjectTagSerializer`.
The serializer field might be named incorrectly and not match any attribute or key on the `list` instance.
Original exception text was: 'list' object has no attribute 'get_lineage'.
```

Reply:
> FYI even with the Meta.model set, I still get an error in the DRF UI.
> But I'm unable to reproduce this issue in tests, and so maybe we don't care?

Ah, that was weird but I figured out the error occurs when the UI tries to render the _form_ for this API endpoint, not when using the endpoint itself. To fix it, I just had to mark the `lineage` field of the serializer as `read_only=True`.